### PR TITLE
fix: booking list page apply

### PR DESCRIPTION
**Pull request by rajasekarcbt**
1. Booking Information Page: I provided the booking information here.
2. Booking Page: Check out the booking details and confirmation on these pages.

I have added styles to the pages mentioned above.

Screenshot from 2024-07-03 21-42-30

Screenshot from 2024-07-03 21-41-57

### CODE REVIEW

**Comment by rajasekarcbt** [comment]
1. Booking Information Page: I provided the booking information here.
2. Booking Page: Check out the booking details and confirmation on these pages.

I have added styles to the pages mentioned above.

File: backend/frontend/src/components/SelectInput.js [hunk 1]
```diff
@@ -1,34 +1,117 @@
...
+  }, []);
+
+  // Update select2 when value or options change
+  useEffect(() => {
+    $(selectRef.current).val(value).trigger("change");
+  }, [value]);
+
+  useEffect(() => {
+    $(selectRef.current).trigger("change.select2");
+  }, [options, addSlots, removeSlots]);
```
Comment (TharunCBT) [inline backend/frontend/src/components/SelectInput.js]: Nits - Dependency need to be in ascending order.

File: backend/frontend/src/components/SelectInput.js [hunk 2]
```diff
@@ -1,34 +1,117 @@
-import React from "react";
+import React, { useEffect, useRef } from "react";
+import $ from "jquery";
+import "select2/dist/css/select2.min.css";
```
Comment (TharunCBT) [inline backend/frontend/src/components/SelectInput.js:3]: Nits - imports should be arranged properly.

**Review by TharunCBT — APPROVED** [review]

File: backend/frontend/src/components/SelectInput.js (hunk 1 above)
Comment (rajasekarcbt) [reply backend/frontend/src/components/SelectInput.js]: chages are done

**Pull request merged**